yarn-project/sequencer-client/src/config.ts

```diff
@@ -76,5 +79,8 @@ export function getConfigEnvVars(): SequencerClientConfig {
     publisherPrivateKey,
     maxTxsPerBlock: SEQ_MAX_TX_PER_BLOCK ? +SEQ_MAX_TX_PER_BLOCK : 32,
     minTxsPerBlock: SEQ_MIN_TX_PER_BLOCK ? +SEQ_MIN_TX_PER_BLOCK : 1,
+    // TODO: undefined should not be allowed for the following 2 values in PROD
+    coinbase: COINBASE ? EthAddress.fromString(COINBASE) : undefined,
```

Comment:
Unclear if you could just use address zero. If the fees require a refund that is initiated by the fee recipient that one will be a problem. But if not could likely work, but just throwing away all fees and rewards.

Reply:
I think we should do that because if the intention of the sequencer is to burn their rewards they can set coinbase to `0x000000000000000000000000000000000000dEaD` and `feeRecipient` to some other burner value. Allowing for 0 could lead to accidental losses due to misconfiguration.

But I also think that it makes sense to tackle later as currently it makes sense to not bother with proper configs so would just create the issue and link it in here an in `sequencer.ts`. Would you agree?